added scrollable directive to the dashboard grid container

MaddyDev opened the pull request:
Before without scroll:
![image](https://user-images.githubusercontent.com/12754347/60360610-ee11e000-9990-11e9-87b8-b7e658a9f99d.png)

Added scroll:

![image](https://user-images.githubusercontent.com/12754347/60360519-bd31ab00-9990-11e9-8a76-cda3d9cec6b9.png)

alanrenmsft commented:
@MaddyDev , Could you please describe the issue you are trying to solve? and add screenshots for before and after?

MaddyDev commented:
Added this to address:

https://github.com/microsoft/azuredatastudio/issues/5988#issuecomment-503672054

I added the scroll, I am not able take the screenshots with scroll.

MaddyDev commented:
Fix for #5988
